.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  examples-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest] 
+        python-version: [3.7]
+        examples: [southpark-search, object-search]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install jina[cicd,test,http] requests --no-cache-dir
```

Comment:
Do we have specific packages for cicd?

Reply:
`cicd` in used in jina's ci and it has many packages in `extra-requirements`

Reply:
Good to know !!

Reply:
But I removed it and lets see if it works. I think we don't need it